packages/commonwealth/client/scripts/views/pages/CommunityManagement/AdminsAndModerators/AdminsAndModerators.tsx

```diff
@@ -1,7 +1,174 @@
-import React from 'react';
+import { AccessLevel } from '@hicommonwealth/core';
+import axios from 'axios';
+import React, { useEffect, useMemo, useState } from 'react';
+import app from 'state';
+import { useDebounce } from 'usehooks-ts';
+import FeatureHint from 'views/components/FeatureHint';
+import { CWText } from 'views/components/component_kit/cw_text';
+import {
+  APIOrderBy,
+  APIOrderDirection,
+} from '../../../../../scripts/helpers/constants';
+import useSearchProfilesQuery from '../../../../../scripts/state/api/profiles/searchProfiles';
+import RoleInfo from '../../../../models/RoleInfo';
+import { ComponentType } from '../../../components/component_kit/types';
+import { UpgradeRolesForm } from '../../../pages/manage_community/upgrade_roles_form';
+import { ManageRoles } from '../../manage_community/manage_roles';
+import './AdminsAndModerators.scss';
 
 const AdminsAndModerators = () => {
```

Comment:
we should also check 
1. is user logged in
2. is user a community or a site admin
and only then render this page.